include/envoy/runtime/runtime.h

```diff
@@ -71,6 +74,14 @@ class Snapshot {
 
   typedef std::unique_ptr<const OverrideLayer> OverrideLayerConstPtr;
 
+  // Returns true if a deprecated feature is allowed.
+  //
+  // Unlike most runtime features, deprecated features are either allowed or disallowed.
+  // They are allowed by default, or with explicit configuration to "100" via runtime configuration.
```

Comment:
Should we just add a boolean runtime type? It would be quite easy to read in true/false from the filesystem for example as a special string much like we do for integers? This might be useful elsewhere where we don't care about percentages. I don't feel strongly about this but throwing it out there.